test/integration/consul-container/test/gateways/http_route_test.go

```diff
@@ -189,23 +193,21 @@ func TestHTTPRouteFlattening(t *testing.T) {
 	require.NoError(t, cluster.ConfigEntryWrite(routeOne))
 	require.NoError(t, cluster.ConfigEntryWrite(routeTwo))
 
-	//create gateway service
-	gatewayService, err := libservice.NewGatewayService(context.Background(), gatewayName, "api", cluster.Agents[0], listenerPort)
+	// create gateway service
+	gatewayService, err := libservice.NewGatewayService(context.Background(), gatewayName, "api", namespace, cluster.Agents[0], listenerPort)
 	require.NoError(t, err)
-	libassert.CatalogServiceExists(t, client, gatewayName, nil)
+	libassert.CatalogServiceExists(t, client, gatewayName, &api.QueryOptions{Namespace: namespace})
```

Comment:
I _think_ this works okay if `namespace == ""`

Reply:
yup, in OSS the `getNamespace` function always returns an empty string